package.json

```diff
@@ -54,8 +54,8 @@
         "size-limit": "^6.0.3",
         "tslint": "^6.1.3",
         "typescript": "^4.5.4",
-        "webpack": "4.35.0",
-        "webpack-cli": "3.3.5",
+        "webpack": "^5.72.0",
+        "webpack-cli": "^4.9.2",
         "webpack-dev-server": "3.7.2"
     },
```

Comment:
How have you tested that these bundles actually work?

Reply:
Yep, I made a test vanilla html app and pulled in the supertokens, session and emailpassword bundles and tested out the basic flow of email password auth, doesSession exist and signout